pystac/item.py

```diff
@@ -481,3 +481,14 @@ def from_file(cls, href: str, stac_io: Optional[pystac.StacIO] = None) -> "Item"
     @classmethod
     def matches_object_type(cls, d: Dict[str, Any]) -> bool:
         return identify_stac_object_type(d) == STACObjectType.ITEM
+
+    @property
+    def __geo_interface__(self) -> Optional[Dict[str, Any]]:
+        """Returns this item's geometry.
+
+        https://gist.github.com/sgillies/2217756
+
+        Returns:
+            Dict[str, Any]: The Item's geometry
+        """
+        return self.geometry
```

Comment:
should this leak the internal dict?

Reply:
I'm not sure what the expectations for `__geo_interface__` are, but you're probably right that its safer to deepcopy.